idea/jvm-debugger/jvm-debugger-test/test/org/jetbrains/kotlin/idea/debugger/test/AbstractKotlinEvaluateExpressionTest.kt

```diff
@@ -62,9 +64,11 @@ abstract class AbstractKotlinEvaluateExpressionTest : KotlinDescriptorTestCaseWi
 
     private val exceptions = ConcurrentHashMap<String, Throwable>()
 
-    override fun runBare() {
-        // DO NOTHING
-    }
+    open fun fragmentCompilerBackend() = CodeFragmentCompiler.Companion.FragmentCompilerBackend.JVM
+
+//    override fun runBare() {
```

Comment:
Maybe this one can be removed?

Reply:
Do you mean `fragmentCompilerBackend`? Or the clean-up of my accidental comment? :sweat_smile: Because I will definitely do the latter!

Reply:
Oh, sorry, it's clear to me now. The "preview" of your comment on the main conversation page did not show the _removal_ of uncommented runBare, so it looked like your comment began at the declaration of `fragmentCompilerBackend`.

Quick fix incoming!